### DISCUSSION TITLE
polytomies

### DISCUSSION THREAD

**Discussion by frezaei98** [tskit-dev/tsinfer#869] [Q&A] [answered]
Hello, 
 Can polytomies be present in the inferred trees generated by tsinfer?  If not, how does tsinfer ensure no polytomies?

**Comment by hyanwong** [comment, marked as answer]
Hi - yes, polytomies are often generated by tsinfer. See e.g. https://tskit.dev/tsinfer/docs/stable/tutorial.html#toy-example. The `tsqc` utility currently [being developed](https://github.com/tskit-dev/tsqc) will display stats on the number of children for large inferences (we call this the "arity" of a node).

Tskit gives a tree-by-tree way of breaking polytomies in https://tskit.dev/tskit/docs/stable/python-api.html#tskit.Tree.split_polytomies
